pkg/cli/cmd/resourceprovider/create/create.go

```diff
@@ -140,6 +140,7 @@ func (r *Runner) Run(ctx context.Context) error {
 		r.Output.LogInfo("Creating resource type %s/%s", r.ResourceProvider.Name, resourceTypeName)
 		_, err := client.CreateOrUpdateResourceType(ctx, "local", r.ResourceProvider.Name, resourceTypeName, &v20231001preview.ResourceTypeResource{
 			Properties: &v20231001preview.ResourceTypeProperties{
+				Capabilities:      to.SliceOfPtrs(resourceType.Capabilities...),
```

Comment:
Prior to this change, it was possible to include capabilities in a manifest, but they were unused.